README.md

```diff
@@ -50,6 +50,88 @@ The following table lists all available options:
 
 **Note:** Once you've called `initialize`, the configuration will be the same every time you `require('lovetoys.lovetoys')`.
 
+## Quickstart
+```lua
+-- Include the library.
+local lovetoys = require('lovetoys.lovetoys')
+
+-- Initialize:
+-- debug = true will enable library console logs
+-- globals = true will register lovetoys classes in the global namespace
+-- so you can access i.e. Entity() in addition to lovetoys.Entity()
+lovetoys.initialize({globals = true, debug = true})
+
+function love.load()
+    -- Define a component class.
+    local Position = Component.create("position", {"x", "y"}, {x = 0, y = 0})
```

Comment:
Please also use an uppercase name here and later on, to be consistent with the "Components as classes" concept.
thanks!